### DISCUSSION TITLE
Cannot open/modify files after installing and integrating Channels to my web server

### DISCUSSION THREAD

**Discussion by kalideir** [django/channels#1653] [Help!] [answered]
After I integrated Channels, Redis to my Django server I noticed I cannot access files inside my HTTP views.
For instance, 

`df = pd.read_csv(full_path, etc)`

I am working on Windows 10, pipenv environment, 
Django@3.1.7
Python@3.8.x
Channels@3.0.3
Channels-Redis@3.2.0

Once I remove the asgi declaration line and channels from the settings.py file everything works as expected.

**Comment by yedpodtrzitko** [comment]
Hi,
can you elaborate a bit more what does "cannot access files" mean? Is it throwing an exception? If so, can you provide the traceback? Can you also provide more code than a single line without context (ie. is it inside async function etc.)? Thanks.

**Reply by kalideir** [reply]
Yes indeed, it was raising file used by another process error. What I did is add Redis channel layer, restart pc and it was gone.

**Comment by kalideir** [comment, marked as answer]
[Answer](https://github.com/django/channels/discussions/1653#discussioncomment-450186)
